docs/EN_US/HPCCClientTools/CT_Mods/ECLCC.xml

```diff
@@ -517,6 +516,13 @@
                 program.</para></entry>
               </row>
 
+              <row>
+                <entry>--expand &lt;path&gt; </entry>
+
+                <entry>Expand the contents of an archive to the specified
+                directory </entry>
```

Comment:
Not yet implemented, but the contents of the submitted query will also output to stdout.  (HPCC-19709)  Probably worth including that in this PR, even thought the software hasn't implemented it.